css-transforms-2/Overview.bs

```diff
@@ -1106,6 +1106,65 @@ The transform functions <<matrix()>>, ''matrix3d()'' and ''perspective()'' get c
 
 For interpolations with the primitive ''rotate3d()'', the direction vectors of the transform functions get normalized first. If the normalized vectors are equal, the rotation angle gets interpolated numerically. Otherwise the transform functions get converted into 4x4 matrices first and interpolated as defined in section <a href="#matrix-interpolation">Interpolation of Matrices</a> afterwards.
 
+Addition and accumulation of transform lists {#combining-transform-lists}
+============================================
+
+<a lt="value addition">Addition</a> of transform lists is
+defined as <a spec=infra>list</a> concatenation
+such that <var ignore>V<sub>result</sub></var> is equal to
+<var ignore>V<sub>b</sub></var> <a spec=infra for=list>appended</a> to
```

Comment:
Where do these `b` and `a`s come from? (The fact that you had to `ignore` them is a warning sign - they should be defined as inputs, *then* used. Wrap it in a `<div algorithm="transform list addition">` too, so their use is scoped to this algorithm and won't spuriously be counted as uses of those variables elsewhere.

Reply:
They come from here: https://drafts.csswg.org/css-values-4/#combining-values
How do I define them as inputs?

Reply:
Just in prose, like:

> Addition of two transform lists |a| and |b| is defined as...

I don't *yet* have definition ability for algorithm arguments, but for now making the inputs clear in prose will do the job.